--- a/scapy/arch/__init__.py
+++ b/scapy/arch/__init__.py
@@ -45,7 +45,7 @@
 
 LINUX=sys.platform.startswith("linux")
 OPENBSD=sys.platform.startswith("openbsd")
-FREEBSD=sys.platform.startswith("freebsd")
+FREEBSD = 'freebsd' in sys.platform
 NETBSD = sys.platform.startswith("netbsd")
 DARWIN=sys.platform.startswith("darwin")
 SOLARIS=sys.platform.startswith("sunos")
--- a/scapy/arch/unix.py
+++ b/scapy/arch/unix.py
@@ -16,7 +16,7 @@
 import scapy.arch
 
 scapy.config.conf.use_pcap = 1
-scapy.config.conf.use_dnet = 1
+scapy.config.conf.use_dnet = 0
 from pcapdnet import *
 
 
