From: Sophie Brun <sophie@offensive-security.com>
Date: Tue, 29 Sep 2020 11:08:43 +0200
Subject: Change path in test for autopkgtest

Forwarded: not-needed
---
 test/regression.uts | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/test/regression.uts b/test/regression.uts
index cf21ecb..f555e64 100644
--- a/test/regression.uts
+++ b/test/regression.uts
@@ -1469,11 +1469,10 @@ raw(RandASN1Object())
 # RSN Information that isn't parsed properly,
 # causing the SSID to be overridden.
 # This test checks the SSID is parsed properly.
-print(os.path.dirname(__file__))
+print(os.getcwd())
 filename = os.path.abspath(
     os.path.join(
-        os.path.dirname(__file__),
-        "..",
+        os.getcwd(),
         "test",
         "pcaps",
         "bad_rsn_parsing_overrides_ssid.pcap"
