--- a/scapy/arch/linux.py
+++ b/scapy/arch/linux.py
@@ -69,7 +69,7 @@
 
 LOOPBACK_NAME="lo"
 
-with os.popen("tcpdump -V 2> /dev/null") as _f:
+with os.popen("/usr/sbin/tcpdump -V 2> /dev/null") as _f:
     if _f.close() >> 8 == 0x7f:
         log_loading.warning("Failed to execute tcpdump. Check it is installed and in the PATH")
         TCPDUMP=0
--- a/scapy/config.py
+++ b/scapy/config.py
@@ -56,11 +56,11 @@
 
     
 class ProgPath(ConfClass):
-    pdfreader = "acroread"
+    pdfreader = "xpdf"
     psreader = "gv"
     dot = "dot"
     display = "display"
-    tcpdump = "tcpdump"
+    tcpdump = "/usr/sbin/tcpdump"
     tcpreplay = "tcpreplay"
     hexedit = "hexer"
     wireshark = "wireshark"
