--- a/scapy/config.py
+++ b/scapy/config.py
@@ -65,7 +65,7 @@
     psreader = "open" if DARWIN else "xdg-open"
     dot = "dot"
     display = "display"
-    tcpdump = "tcpdump"
+    tcpdump = "/usr/sbin/tcpdump"
     tcpreplay = "tcpreplay"
     hexedit = "hexer"
     tshark = "tshark"
--- a/scapy/utils6.py
+++ b/scapy/utils6.py
@@ -246,7 +246,7 @@
     Extract the MAC address from a modified EUI-64 constructed IPv6
     address provided and use the IANA oui.txt file to get the vendor.
     The database used for the conversion is the one loaded by Scapy,
-    based on Wireshark (/usr/share/wireshark/wireshark/manuf)  None
+    based on Wireshark (/usr/share/wireshark/manuf)  None
     is returned on error, "UNKNOWN" if the vendor is unknown.
     """
     mac = in6_addrtomac(addr)
